stories/Molecules/FooterNavigation/FooterConditions2/footerCondition2.scss

```diff
@@ -27,14 +27,23 @@
       @extend %img_none;
 
       border-bottom: 0;
+      color: $color-white;
       font-size: $font-size-16;
       line-height: 1.4;
+
+      &:hover {
+        color: $color-blue-100;
+      }
     }
   }
 
-  &.white {
+  &.inverted {
     a {
-      color: $color-white;
+      color: $color-black;
+
+      &:hover {
+        color: $color-gray-600;
+      }
```

Comment:
@Shubham-Sharma-77 Seems like we have similar code in footerCondition.scss file so can we create the placeholders for similar code